sdk/identity/identity/tsconfig.samples.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Is this file necessary? `samples/tsconfig.json` is the only thing that extends this, so it seems like there's no reason not to just take these settings and roll them into one tsconfig.

Reply:
I'll look into it!

Reply:
I'll move the extra properties to the tsconfig.json in the samples folder!